InferenceSystem/deploy-aci.yaml

```diff
@@ -12,6 +12,8 @@ properties:
         secureValue: '<cosmos_primary_key>'
       - name: AZURE_STORAGE_CONNECTION_STRING
         secureValue: '<storage_connection_string>'
+      - name: GITHUB_ACCESS_TOKEN
```

Comment:
@micya @DhananjayPurohit good catch on the access token but I think this should go in its own createPR.yaml or something.
 
 The deploy-aci.yaml strings are for deployment to ACI so do not need a GITHUB_ACCESS_TOKEN.

Reply:
`auto-retraining-on-neg-samples.yaml`

Reply:
Sorry for the confusion